composer.json

```diff
@@ -25,7 +25,7 @@
   },
   "require-dev": {
     "phpunit/PHPUnit": "~4.6",
-    "squizlabs/php_codesniffer": "~2.0"
+    "squizlabs/php_codesniffer": "^2.3.1"
```

Comment:
Yea they didn't follow semantic versioning about to raise feature number when new sniffs are added